generic-styles/content-baked.less

```diff
@@ -366,6 +366,10 @@ h1.example-title .text {
   img {
     display: block;
   }
+
+  [data-type="exercise"]:first-of-type {
```

Comment:
you said you don't like this, why is it necessary and why don't you like it?

Reply:
I'm looking through the books for different noteish elements and find some workaround to fix the disabled margin collapsing problem.
This selector is fixing the `Checkpoint 1.18 ` in https://rex-web-issue-1637-ugrznxqolyj.herokuapp.com/books/calculus-volume-3/pages/1-5-conic-sections